docs/developer-guide/etendo-rx/how-to-guides/how-to-filter-for-active-or-inactive.md

```diff
@@ -0,0 +1,35 @@
+## You can filter any request by active or all rows
+
+The **\_noActiveFilter** parameter can be used in any request method to filter active records or all records (active and inactive). This way it allows more control when obtaining the results.
+
+---
+
+## Example of GET request for all records
```

Comment:
When you have options you can group them using tabs, like this example: 
[Grouping code blocks](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks)